src/main/java/org/sonarlint/intellij/actions/SonarLintToolWindow.java

```diff
@@ -469,5 +473,20 @@ private Content getTaintVulnerabilitiesContent() {
   @Override
   public void bindingChanged(@Nullable ProjectBinding previousBinding, @Nullable ProjectBinding newBinding) {
     runOnUiThread(project, this::refreshViews);
+
+    showSharedConfigurationNotification(project, "Project successfully bound with \"" +
+        newBinding.getProjectKey() + "\" on \"" + newBinding.getConnectionName() +
+      "\"." +
+        " If you share this configuration, a file will be created in this working directory," +
+        " making it easier for other team members to configure the binding for the same project." +
+        "You may also decide to share this configuration later from your list of bound projects.", "ShareConfigurationAction",
```

Comment:
For the `doNotShowAgainId` you can use a const with the following format:
`private const val SKIP_SHARED_CONFIGURATION_DIALOG_PROPERTY = "SonarLint.shareConfiguration"`